about.html

```diff
@@ -119,6 +120,7 @@ <h2 class="previous-course_title">Our academic history</h2>
                 </li>
             </ul>
         </section>
```

Comment:
[REQUIRED] Kindly fix as suggested in the images below:
![image](https://user-images.githubusercontent.com/62938735/205289750-a042646d-61fd-4760-a670-2a42fa50469d.png)


![image](https://user-images.githubusercontent.com/62938735/205290070-19aed462-43bd-4830-84af-3cffaa819e78.png)